scripts/train/yamls/finetune/gpt2-arc-easy--cpu-streaming-dataset.yaml

```diff
@@ -0,0 +1,80 @@
+max_seq_len: 512
```

Comment:
Nit (filename): two double dashes `--` before `CPU.` Is that intentional?

Reply:
I just copied from [this file](https://github.com/mosaicml/llm-foundry/blob/105f76631d7f74edebc187f3427008ce4506ee85/scripts/train/finetune_example/gpt2-arc-easy--cpu.yaml), and  append `-streaming-dataset` to the filename. Let me just use one dash.